draft-ietf-acme-acme.md

```diff
@@ -1972,6 +1997,11 @@ domain by verifying that the resource was provisioned as expected.
 5. Verify that key authorization provided by the server matches the token for
    this challenge and the client's account key.
 
+The server MAY follow redirects when dereferencing the URI.
+
+It is RECOMMENDED that the server dereference the URI multiple times from
+various network perspectives, in order to make MitM attacks harder.
```

Comment:
Fair enough. Will break this out into a separate PR, and clarify that the time is not indeterminate, but merely until the authz transitions to valid or invalid.